servers/rendering_server.cpp

```diff
@@ -865,8 +865,22 @@ Error RenderingServer::mesh_create_surface_data_from_arrays(SurfaceData *r_surfa
 					ERR_FAIL_V(ERR_INVALID_DATA);
 				} break;
 			}
-
 			ERR_FAIL_COND_V(array_len == 0, ERR_INVALID_DATA);
+		} else if (i == RS::ARRAY_BONES) {
+			switch (p_arrays[i].get_type()) {
+				case Variant::PACKED_INT32_ARRAY: {
+					Vector<Vector3> v3 = p_arrays[RS::ARRAY_VERTEX];
+					Vector<float> bones = p_arrays[i];
```

Comment:
probably meant to use Vector<uint32_t> here

Reply:
Changed to Vector<int32> as used in other places for RS::ARRAY_BONES